src/main/java/io/appium/java_client/TouchAction.java

```diff
@@ -118,13 +168,16 @@ public TouchAction moveTo(WebElement el) {
      * @param x change in x coordinate to move through.
      * @param y change in y coordinate to move through.
      * @return this TouchAction, for chaining.
+     * @deprecated {@link #moveTo(ActionOptions)} instead
      */
-    public TouchAction moveTo(int x, int y) {
-        ActionParameter action = new ActionParameter("moveTo");
-        action.addParameter("x", x);
-        action.addParameter("y", y);
+    @Deprecated
+    public T moveTo(int x, int y) {
+        ActionParameter action = new ActionParameter("moveTo",
+                new MoveToOptions()
+                        .withRelativeOffset(x, y));
```

Comment:
@mykola-mokhnach Isn't it absoluteOffset?

Reply:
> * @param x change in x coordinate to move through.
> * @param y change in y coordinate to move through.

That is why I find the current approach useful, because it helps to avoid such misunderstandings while settings options ;)

Reply:
@mykola-mokhnach haha makes sense.thanks :)